src/DocBlock/TypeExpression.php

```diff
@@ -88,7 +88,7 @@ final class TypeExpression
                     (?i)
```

Comment:
I would love to hear more about the purpose of this PR.
looking only at \\\\\\\'"\\'"\'"|"|"\'"\\'\\\\\""\"\""\\\"" is not helping ;)

Reply:
1. escaping the quote character was previously not possible
2. `\n` could be possibly reprinted wrongly, thus tested ;-)
3. the test has 2 types, if parsed, they are parsed as two types - this is what the test does